src/ziflist.c

```diff
@@ -275,9 +275,28 @@ s_reload (ziflist_t *self, bool ipv6)
     DWORD rc = GetAdaptersAddresses (AF_INET, GAA_FLAG_INCLUDE_PREFIX, NULL, NULL, &addr_size);
     assert (rc == ERROR_BUFFER_OVERFLOW);
 
-    PIP_ADAPTER_ADDRESSES pip_addresses = (PIP_ADAPTER_ADDRESSES) zmalloc (addr_size);
-    rc = GetAdaptersAddresses (AF_INET,
-                               GAA_FLAG_INCLUDE_PREFIX, NULL, pip_addresses, &addr_size);
+    static const int maxTries = 3;
```

Comment:
This means it will do only 3 tries for the lifetime of the process. Is this what you intended?